File: src/primus/pawel/Exercise_175.java
package primus.pawel;

import java.util.Random;

// HTML dynamic color string generation
// 6kyu
// https://www.codewars.com/kata/56f1c6034d0c330e4a001059/train/java
// 2021-07-11 T:18:08:04
class Exercise_175 {
    public static String generateColor(Random r) {
        StringBuilder result = new StringBuilder("#");
        for(int i = 0 ; i < 6; i++){
            result.append(Integer.toHexString(r.nextInt(16)));
        }
        return result.toString();
    }

    public static void main(String[] args) {
        System.out.println(generateColor(new Random()));
    }

}

File: src/JS/Exercise_175.js
// Limit string length - 1
// 7kyu
// https://www.codewars.com/kata/5208fc3cb613bc725f000142/train/javascript
// 2022-07-21 T:07:37:43#

function solution(string, limit) {
    return string.length <= limit ? string : string.substring(0, limit) + '...';
}

console.log(solution('Test', 8))